wpengine/faustjs
# Fix Faust.js WordPress Plugin Preview

## Tasks


## Description

These changes include a proposed fix for https://github.com/wpengine/faustjs/issues/1561.

<img width="1698" alt="wp-admin block editor interface showing preview button locations" src="https://github.com/wpengine/faustjs/assets/6676674/1d300657-a3be-462d-b51a-e5f070c6d486">

## Testing

1. Checkout this branch that includes changes to the **faustwp** plugin: `MERL-1205-refactor-custom-preview`
2. Using the updated **faustwp** plugin, create a new post & save as draft (do not publish)
3. Observe that both preview links correctly resolve to display draft content
